php/libraries/NDB_Page.class.inc

```diff
@@ -569,6 +569,7 @@ class NDB_Page
                   $baseurl . '/js/polyfiller.js',
                   $baseurl . '/bootstrap/js/bootstrap.min.js',
                   $baseurl . '/js/components/react.breadcrumb.js',
+                  $baseurl . '/js/components/Form.js',
```

Comment:
doesn't this only need to go in NDB_Form and NDB_MenuFilterForm? Arbitrary pages don't necessarily have forms.